static/includes/TNHardwareGuide.md

```diff
@@ -188,6 +188,9 @@ You might also consider adding a hot-spare drive with any pool configuration, wh
 
 ### Storage Device Burn-In
 
+{{< hint type=important >}}
+**Warning:** After TrueNAS SCALE 22.10 (Bluefin), if you use "admin" for administrasive account to login Web-UI, add "sudo" in shell.
```

Comment:
We don't use double quotes to emphasize text. making a small change this suggested improvement.